inyoka_theme_default/static/style/inyoka/inyoka.less

```diff
@@ -231,6 +236,12 @@ input[type="text"], input[type="email"], input[type="password"], select {
   .form-control-focus();
 }
 
+table {
```

Comment:
hm, is it possible to rebase the commit, so that it only contains the changes in macros.html?